hudi-common/src/main/java/org/apache/hudi/common/model/debezium/AbstractDebeziumAvroPayload.java

```diff
@@ -76,7 +75,12 @@ public AbstractDebeziumAvroPayload(Option<GenericRecord> record) {
     boolean delete = false;
     if (insertRecord instanceof GenericRecord) {
       GenericRecord record = (GenericRecord) insertRecord;
-      Object value = record.get(DebeziumConstants.FLATTENED_OP_COL_NAME);
+      Object value;
+      if (record.getSchema().getField(DebeziumConstants.FLATTENED_OP_COL_NAME) == null) {
+        value = null;
+      } else {
+        value = record.get(DebeziumConstants.FLATTENED_OP_COL_NAME);
+      }
```

Comment:
the new check does not look clean; it looks like whenever we try to get a column's value, we need to check its nullability? are we able to retrieve the value the without this kind of check